.github/workflows/docker-build.yaml

```diff
@@ -45,21 +45,21 @@ jobs:
         # Setup docker build to use your Namespace workspace builder
         name: Set up Namespace Buildx
         uses: namespacelabs/nscloud-setup-buildx-action@v0
-      - name: Build runwasi as static muls for amd64 and arm64
-        run: |
-          git clone https://github.com/containerd/runwasi.git &&
-          cd runwasi &&
-          docker build --platform=linux/arm64,linux/amd64 --build-arg BASE_IMAGE=alpine --build-arg CRATE="containerd-shim-wasmtime,containerd-shim-wasmedge,containerd-shim-wasmer" -t nscr.io/a8fcp47vcfori/build-runwasi --push . &&
-          cd -
+#      - name: Build runwasi as static muls for amd64 and arm64
```

Comment:
Is this due to issue described here: https://github.com/containerd/runwasi/issues/328?

Reply:
Not directly, there are two reasons

- I wanted to use the latest changes before release
- The 0.1.1 release needs libc which breaks Arch Linux support